examples/meta/src/base_api/factory.sg

```diff
@@ -0,0 +1,13 @@
+Machine lib_svm = machine("LibSVM")
+Machine lda = machine("LDA")
+Kernel kernel_gaussian = kernel("GaussianKernel")
+Kernel kernel_linear = kernel("LinearKernel")
+
+RealMatrix real_matrix(2,2)
+Features features_from_matrix = features(real_matrix)
+
+File features_file = csv_file("../../data/classifier_binary_2d_nonlinear_features_train.dat")
+Features features_from_file = features(features_file, enum EPrimitiveType.PT_FLOAT64)
+
+File labels_file = csv_file("../../data/classifier_binary_2d_nonlinear_labels_train.dat")
```

Comment:
labels from file will be `CDenseLabels` which are then converted in the classes (needs some refactor in algorithms, see below)

Reply:
if you have just added the factory.sg could you change the relative path (`../../`) to an env var? i'm happy to add the support for that for every language 💃

Reply:
as if we install every these meta examples, the location of data should be somewhere very else :)

Reply:
Yep absoltely agree. But I think maybe let's do that for all meta examples in another PR?
I can do that next